### ISSUE TITLE
Will the script survive Proxmox upgrades?

### ISSUE THREAD

**Issue by logicito** [Meliox/PVE-mods#70]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Sensors output**
If applicable, provide sensor datadump (use the argument save-sensors-data)

**Comment by Meliox** [comment]
I think you forgot the question apart from what can be read in the header...
No, it will not survive an update. I believe the package is pve-manager. If that package is updated the script needs to be reinstalled. Please note that proxmox may introduce breaking chances at any time and thus is can break the script.

**Comment by logicito** [comment]
Thank you, I appreciate the quick response

**Comment by divStar** [comment]
Just checked with Proxmox VE 8.2.9 (it just had a `pve-manager` update, too) and indeed the script had to be re-applied, but it worked without any issues. I even wrote a Terraform configuration project to prepare my host for further use (I haven't shared it yet), but in that project it downloads the anti-nag script and installs it upon applying the configuration and uninstalls it upon destroying the configuration (amongst many other things).

TL;DR: works great, thanks!

P.s.: I am not sure, but perhaps it'd be possible to create some recurring way of automatically applying the script, but since if the script fails the whole UI fails, I wouldn't really suggest it.
